packages/smooth_app/lib/pages/folksonomy/folksonomy_card.dart

```diff
@@ -0,0 +1,143 @@
+      child: Card(product),
+    );
+  }
+}
+
+class Card extends StatelessWidget {
+  const Card(this.product);
+  final Product product;
+
+  @override
+  Widget build(BuildContext context) {
+    final AppLocalizations appLocalizations = AppLocalizations.of(context);
+
+    return InkWell(
+      onTap: () async {
+        if (!await ProductRefresher().checkIfLoggedIn(
+          context,
+          isLoggedInMandatory: true,
+        )) {
+          return;
+        }
+        if (!context.mounted) {
+          return;
+        }
+
+        Navigator.of(context)
+            .push(
+              MaterialPageRoute<void>(
+                builder: (BuildContext context) => FolksonomyPage(product),
+              ),
+            )
+            .then((void completion) => <Set<Future<void>>>{
+                  if (context.mounted)
+                    <Future<void>>{
+                      Provider.of<FolksonomyProvider>(context, listen: false)
+                          .fetchProductTags(),
+                    }
+                });
+      },
+      child: Padding(
+        padding: const EdgeInsets.only(top: LARGE_SPACE),
+        child: buildProductSmoothCard(
+          title: Stack(
+            children: <Widget>[
+              Center(child: Text(appLocalizations.product_tags_title)),
+            ],
+          ),
+          body: Container(
+            width: double.infinity,
+            padding: const EdgeInsetsDirectional.all(LARGE_SPACE),
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              mainAxisAlignment: MainAxisAlignment.start,
+              children: <Widget>[
+                CardList(),
+              ],
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+class CardList extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    final AppLocalizations appLocalizations = AppLocalizations.of(context);
+    final FolksonomyProvider provider = context.watch<FolksonomyProvider>();
+
+    if (provider.isLoading) {
+      return const Center(child: CircularProgressIndicator.adaptive());
+    } else if (!provider.tagsExist) {
+      return Center(
+        child: Row(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: <Widget>[
+            Expanded(
+              child: Text(
+                appLocalizations.no_product_tags_found_message,
+                textAlign: TextAlign.center,
+              ),
+            ),
+            Icon(ConstantIcons.instance.getForwardIcon()),
+          ],
+        ),
+      );
+    } else {
+      final Map<String, ProductTag> tags = provider.productTags!;
+      final Iterable<MapEntry<String, ProductTag>> displayTags =
+          tags.entries.toList().take(5);
+
+      return Padding(
+        padding: const EdgeInsetsDirectional.only(
+          top: VERY_SMALL_SPACE,
+          bottom: VERY_SMALL_SPACE,
+        ),
+        child: Semantics(
+          button: true,
```

Comment:
`Semantics` are for accessibility and if you want to override it.
Is-it really what you want to do?